presto-main/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -1242,6 +1241,16 @@ private boolean canResolveOperator(OperatorType operatorType, List<TypeSignature
         }
     }
 
+    private ConnectorTableLayoutHandle getTableLayout(Session session, TableHandle tableHandle)
```

Comment:
It looks like this method is not used in this commit. Let's remove it from here.